create-artists/create-artists.py

```diff
@@ -0,0 +1,21 @@
+import csv
+from os.path import exists
+from pathlib import Path
+
+artists = set()
+
+path = Path(__file__).parent / "../csvs/card.csv"
+with path.open(newline='') as csvfile:
+    reader = csv.reader(csvfile, delimiter='\t', quotechar='"')
+    next(reader)
+
+    for row in reader:
+        artists.add(row[2])
```

Comment:
This seems to be pulling not from the artist column, but from the name column. But generating the list seemed to work great, so if you have it pull from the right column I'll re-test!